packages/usa-pagination/src/styles/_usa-pagination.scss

```diff
@@ -127,7 +127,7 @@ $pagination-hover-token: list.nth($pagination-link-tokens, 2);
   display: inline-flex;
   justify-content: center;
   padding: units($pagination-margin-padding);
-  text-decoration: none;
+  text-decoration: underline;
   width: 100%;
 
```

Comment:
**issue:** This is my mistake: I advised that we add `text-decoration: underline`, but we can probably just remove `text-decoration` styles completely from the element since it has baseline underline styles. Can you remove the `text-decoration` style declaration here?